icons/replit/replit-original-wordmark.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="M128 59.617V57.77c0-.315-.255-.558-.583-.558h-6.313v-3.578a.563.563 0 0 0-.582-.56h-2.401a.574.574 0 0 0-.583.56v3.577h-3.729a.576.576 0 0 0-.583.56v1.845c0 .318.255.56.583.56h3.729v5.48c0 4.193 1.928 6.132 6.078 6.132h3.383a.574.574 0 0 0 .582-.56v-1.844c0-.318-.254-.56-.582-.56h-2.838c-2.256 0-3.057-.913-3.057-3.187v-5.442h6.313c.328-.02.583-.28.583-.578Zm-18.977-8.034h-3.275a.574.574 0 0 0-.583.56v2.366c0 .318.256.56.583.56h3.275a.574.574 0 0 0 .582-.56v-2.366a.602.602 0 0 0-.182-.399.573.573 0 0 0-.4-.16Zm.437 17.242V57.77c0-.316-.255-.559-.583-.559h-7.332a.574.574 0 0 0-.582.56v1.844c0 .318.254.56.582.56h4.349v8.648h-4.35a.56.56 0 0 0-.402.158.583.583 0 0 0-.178.401v1.845c0 .316.253.56.58.56h12.373a.575.575 0 0 0 .583-.56v-1.845c0-.317-.255-.559-.583-.559l-4.457.002Zm-11.245 0h-3.71V52.142a.562.562 0 0 0-.582-.56h-7.278a.574.574 0 0 0-.582.56v1.844c0 .318.254.56.582.56h4.293v14.278h-4.821a.574.574 0 0 0-.583.559v1.844c0 .317.256.56.583.56h12.1a.574.574 0 0 0 .582-.56v-1.844c0-.318-.256-.56-.584-.56Zm-19.76-11.893c-1.236 0-2.128.28-2.819.95l-.856.84h-.782l-.291-1.08a.604.604 0 0 0-.213-.31.58.58 0 0 0-.35-.12h-1.639a.574.574 0 0 0-.582.56v19.085c0 .317.255.56.582.56h2.403a.574.574 0 0 0 .582-.56V72.72l-.31-2.219.8-.186.855.84c.674.633 1.493.949 2.711.949 3.53 0 5.66-2.497 5.66-7.586-.018-5.088-2.13-7.586-5.75-7.586Zm-1.182 12.116c-1.8 0-2.802-.95-2.802-2.76v-3.577c0-1.808.984-2.759 2.802-2.759 2.201 0 3.203 1.417 3.203 4.53-.018 3.149-1.002 4.566-3.203 4.566Zm-16.52-12.116c-4.586 0-7.043 2.498-7.043 7.568 0 5.09 2.439 7.586 6.952 7.586 3.492 0 5.75-1.49 6.35-4.044.09-.354-.2-.69-.565-.69h-2.183a.586.586 0 0 0-.327.102.607.607 0 0 0-.219.27c-.382 1.1-1.474 1.622-2.948 1.622-2.292 0-3.384-1.286-3.53-4.288h9.953c.546-5.608-1.727-8.126-6.44-8.126Zm-3.404 5.853c.347-2.199 1.42-3.224 3.385-3.224 2.093 0 3.076 1.136 3.13 3.224h-6.515Zm-10.734-4.92-.82.84h-.818l-.254-1.064c-.054-.26-.29-.429-.564-.429h-5.24a.574.574 0 0 0-.583.56v1.844c0 .318.255.56.583.56h3.093v8.648h-3.603a.574.574 0 0 0-.582.56v1.844c0 .317.255.56.582.56h11.336a.573.573 0 0 0 .581-.56v-1.844a.562.562 0 0 0-.581-.56h-4.167v-6.113c0-1.715.983-2.61 2.911-2.61h3.13a.575.575 0 0 0 .582-.558V57.51c0-.318-.255-.56-.582-.56h-1.856c-1.365-.019-2.439.242-3.148.914Z" fill="#F5F9FC"/><path d="M0 50.938c0-.514.2-1.007.554-1.37A1.869 1.869 0 0 1 1.891 49h8.827c.501 0 .982.204 1.337.568.355.363.554.856.554 1.37v8.395H1.891a1.87 1.87 0 0 1-1.337-.567A1.962 1.962 0 0 1 0 57.396v-6.459Zm12.609 8.395h10.718c.501 0 .982.205 1.337.568s.554.856.554 1.37v6.458c0 .514-.2 1.007-.554 1.37a1.87 1.87 0 0 1-1.337.568H12.609V59.333ZM0 71.604c0-.514.2-1.007.554-1.37a1.869 1.869 0 0 1 1.337-.567H12.61v8.395c0 .514-.2 1.007-.554 1.37a1.869 1.869 0 0 1-1.337.568H1.89a1.87 1.87 0 0 1-1.337-.567A1.962 1.962 0 0 1 0 78.063v-6.459Z" fill="#F26207"/></svg>
```

Comment:
Could you also make a `plain-wordmark` version based on this one please? Make sure it adheres to our [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) :)

Reply:
> Could you also make a `plain-wordmark` version based on this one please? Make sure it adheres to our [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) :)

Sure, I will.